crypto/cmp/cmp_server.c

```diff
@@ -376,6 +416,61 @@ static OSSL_CMP_MSG *process_certConf(OSSL_CMP_SRV_CTX *srv_ctx,
     return msg;
 }
 
+/* pollreq should be handled sepreately, to avoid recursive call */
```

Comment:
I suppose it would be possible to prevent wrong recursion als in other ways,
but certainly good to extract this code portion from the lengthy `OSSL_CMP_SRV_process_request` function body .